### DISCUSSION TITLE
What versions of Minecraft does phantom work with?

### DISCUSSION THREAD

**Discussion by jhead** [jhead/phantom#168] [Q&A] [answered]
Will it work with the latest version?

**Comment by jhead** [comment, marked as answer]
**Phantom has been tested with Minecraft version `1.8` up to `1.19` currently.**

Usually, no update to phantom is needed when a new version of Minecraft comes out, but the server you're connecting to still needs to update!

**Comment by GoCards2009** [comment]
Is there going to be support for switch, and if not is there another repository that can do the same thing but for switch?
